src/contacts/model/ContactModel.ts

```diff
@@ -14,12 +14,14 @@ import { cleanMailAddress } from "../../api/common/utils/CommonCalendarUtils.js"
 import { ContactListInfo } from "../view/ContactListViewModel.js"
 import { GroupInfo, GroupInfoTypeRef, GroupMembership, UserTypeRef } from "../../api/entities/sys/TypeRefs.js"
 import { EntityEventsListener, EntityUpdateData, EventController, isUpdateForTypeRef } from "../../api/main/EventController.js"
+import Stream from "mithril/stream"
+import stream from "mithril/stream"
 
 assertMainOrNode()
 
 export class ContactModel {
 	private contactListId: LazyLoaded<Id | null>
-	private contactListInfo: ContactListInfo[] = []
+	private contactListInfo: Stream<ReadonlyArray<ContactListInfo>> = stream()
```

Comment:
should probably be initialized with `[]`

Reply:
we use it below to check whether we already loaded it or not